WordPress/Classes/ViewRelated/Blog/My Site/MySiteViewController+FAB.swift

```diff
@@ -32,6 +32,16 @@ extension MySiteViewController {
         if blog?.supports(.pages) ?? false {
             actions.append(PageAction(handler: newPage, source: source))
         }
+        if UITestConfigurator.isEnabled(.screenshotGeneration), let blog {
+            actions.append(AnonymousAction(title: "create-mock-post") {
+                let post = blog.createDraftPost()
+                post.postTitle = "Welcome to Gutenberg"
+                post.content = """
```

Comment:
I relied on `WireMock` in Reader, but then switched to mocks in code for screenshots. For this purpose, we are not interested in testing the flows end-to-end, so there is no reason to mock the JSONs.